libZipSharp.dll.config

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+	<dllmap dll="libzip.dll" os="osx" target="libzip.4.dylib"/>
+	<dllmap dll="libzip.dll" os="linux" target="libzip.so"/>
```

Comment:
It would be better if it used `libzip.so.4` as the target. On Linux the `.so` file will be present only if development packages are installed.